src/state/migrations/__test__/1.test.ts

```diff
@@ -16,17 +13,32 @@ const MOCK_EVM_SIGNATURE = 'fake_signature_woohoo';
 const MOCK_SOLANA_ADDRESS = '4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU';
 const MOCK_DYDX_ADDRESS = 'dydx1';
 
-const MOCK_EVM_WALLET_INFO: WalletInfo = {
+enum CosmosWalletType {
+  KEPLR = 'keplr',
+}
+
+enum WalletType {
+  CoinbaseWallet = 'COINBASE_WALLET',
+  Phantom = 'PHANTOM',
+}
+
+enum ConnectorType {
+  Coinbase = 'coinbase',
+  Cosmos = 'cosmos',
+  PhantomSolana = 'phantomSolana',
+}
```

Comment:
is it possible to use `WalletType` and `ConnectorType` from `1.ts`? I think it should be fine to import from the migration file

Reply:
yeah I just didn't want to rename and import them and I thought this was fine for a test....what if I just inline the strings?

Reply:
okay pushed that